staging/src/k8s.io/csi-translation-lib/plugins/aws_ebs.go

```diff
@@ -45,8 +45,21 @@ func NewAWSElasticBlockStoreCSITranslator() InTreePlugin {
 	return &awsElasticBlockStoreCSITranslator{}
 }
 
-// TranslateInTreeStorageClassParametersToCSI translates InTree EBS storage class parameters to CSI storage class
+// TranslateInTreeStorageClassToCSI translates InTree EBS storage class parameters to CSI storage class
 func (t *awsElasticBlockStoreCSITranslator) TranslateInTreeStorageClassToCSI(sc *storage.StorageClass) (*storage.StorageClass, error) {
+	params := map[string]string{}
+
+	for k, v := range sc.Parameters {
+		switch strings.ToLower(k) {
+		case "fstype":
+			params["csi.storage.k8s.io/fstype"] = v
```

Comment:
Do you also plan to convert zones to topologies here in the future?

Reply:
Since `zone` and `zones` are deprecated, I would like to not to do these translations if possible. Let me know if you think otherwise @ddebroy @davidz627

Reply:
IMO we need to support the API. A cluster that is updated to migrated version and uses `zone` and `zones` should keep working.

Reply:
Does that mean `zone` and `zones` are not really deprecated? And thinking further, any parameters within storageclass parameters are not deprecateale? If this is the case, the deprecation [here](https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs) is confusing

Reply:
@leakingtapan deprecation does not mean we do not support it anymore - I believe it means we don't recommend usage and that support will be removed in a future release. Therefore until that functionality is "removed" (not sure when) we will need to support it during migration

Reply:
We can argue under which paragraph in deprecation policy it falls, IMO it's part of storage.k8s.io/v1 StorageClass API object. And thus we may remove things from there only after introducing storage.k8s.io/v2.

Reply:
@jsafrane @davidz627 thx for the inputs and they are fair enough. I will add this translation then. But could I open a new PR for the zone/zones translation? since I'm not sure if I have time for it this week since the code freeze is soon.